client/payment-details/transaction-breakdown/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface TransactionDetails {
+	store_amount: number;
+	store_currency: string;
+	customer_amount: number;
+	customer_currency: string;
+	store_fee: number;
+	store_amount_captured: number;
+	customer_amount_captured: number;
+	customer_fee: number;
+}
+
+export interface FeeRate {
+	type: string;
+	additional_type?: string;
+	percentage_rate: number;
+	fixed_rate: number;
+	currency: string;
+}
```

Comment:
To clarify this for code readers who are less familiar (me), it would help to add type annotations here, or even define the expected string values:

```suggestion
export interface FeeRate {
	/** The primary type of the fee, `base` or `additional`. */
	type: string;
	/** Fee sub-category that's only used when type is `additional`. It specifies what kind of additional fee is being applied, e.g. `international`, `fx`, etc. */
	additional_type?: string;
}
```

Reply:
> or even define the expected string values

```suggestion
export interface FeeRate {
	/** The primary type of the fee. */
	type: 'base' | 'additional' | 'discount';
	/** Fee sub-category that's only used when type is `additional`. It specifies what kind of additional fee is being applied, e.g. `international`, `fx`, etc. */
	additional_type?: 'international' | 'fx' | 'wcpay-subscription' | 'device';
	/** Percentage fee rate. */
	percentage_rate: number;
	/** Fixed fee rate. */
	fixed_rate: number;
	/** Currency of the fee rate. */
	currency: string;
}
```

Reply:
Thank you for this great suggestion! I updated types definitions in 350c152d0